tests/integration-tests/tests/createami/test_createami/test_build_image/image.config.with.warnings.yaml

```diff
@@ -0,0 +1,12 @@
+Image:
```

Comment:
What's this config file? Do we use this in our test?

Reply:
Not sure -- I didn't add this (I adopted this work)

Reply:
This will be moved and leveraged in a new test that uses the `suppress-validators` and `dryrun` flags (separate PR)